eng/imports/HostAgnostic.props

```diff
@@ -60,7 +60,7 @@
 
     <!-- 3rd party -->
     <PackageReference Include="Newtonsoft.Json" />
-    <PackageReference Include="IsExternalInit" PrivateAssets="all" />
+    <PackageReference Include="PolySharp" PrivateAssets="all" IncludeAssets="runtime; build; native; contentfiles; analyzers" />
```

Comment:
I wish if there is a document I can refer to learn more about such kind setting more properties or constraint in package reference .

Reply:
I think this document should cover that info: https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files

Let me know if there are specific questions that doc doesn't answer for you.

Note that the change here is based on the recommendation from NuGet.org: https://www.nuget.org/packages/PolySharp/